CHANGELOG.md

```diff
@@ -37,15 +37,17 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## Unreleased
 
-* nothing
+### Improvements
+
+* Add marker address to add marker event [#1499](https://github.com/provenance-io/provenance/issues/1499)
 
 ---
 
 ## [v1.15.0-rc2](https://github.com/provenance-io/provenance/releases/tag/v1.15.0-rc2) - 2023-04-19
 
 ### Features
 
-* Add support for tokens restricted marker sends with required attributes [#1256](https://github.com/provenance-io/provenance/issues/1256))
+* Add support for tokens restricted marker sends with required attributes [#1256](https://github.com/provenance-io/provenance/issues/1256)
```

Comment:
do we want to add this here or rather when we do the back port PR?

Reply:
Oh, I didn't add anything here.  I fixed a double `)` in our changelog.